presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -5875,6 +5875,29 @@ public void testRefreshMaterializedView()
         assertQueryFails("REFRESH MATERIALIZED VIEW test_customer_view_5", ".*mismatched input '<EOF>'\\. Expecting: '\\.', 'WHERE'.*");
     }
 
+    @Test
+    public void testAlphaFormatDdl()
+    {
+        assertUpdate("CREATE TABLE test_table (col1 bigint) WITH (format = 'ALPHA')");
+        assertUpdate("ALTER TABLE test_table ADD COLUMN col2 bigint");
+        assertUpdate("ALTER TABLE test_table DROP COLUMN col2");
+        assertUpdate("DROP TABLE test_table");
+
+        assertUpdate("CREATE TABLE test_partitioned_table (col1 bigint, ds VARCHAR) WITH (format = 'ALPHA', partitioned_by = ARRAY['ds'])");
+        assertUpdate("ALTER TABLE test_partitioned_table ADD COLUMN col2 bigint");
+        assertUpdate("ALTER TABLE test_partitioned_table DROP COLUMN col2");
+        assertUpdate("DROP TABLE test_partitioned_table");
+    }
+
+    @Test
+    public void testAlphaFormatDml()
```

Comment:
Can these tests interfere when run in parallel because they have same table name?

Reply:
Good point! Fixed it by giving each table a unique name.